Robinhood/Robinhood.py

```diff
@@ -188,117 +208,137 @@ def instruments(self, stock):
         """
+
         url = str(self.endpoints['quotes']) + "?symbols=" + ",".join(stocks)
+
         try:
             req = requests.get(url)
             req.raise_for_status()
             data = req.json()
+
         except requests.exceptions.HTTPError:
-            raise NameError('Invalid Symbols: ' + ",".join(stocks)) #TODO: custom exception
+            raise RH_exception.InvalidTickerSymbol()
+
 
         return data["results"]
 
+
     def get_quote_list(self, stock='', key=''):
-        """Returns multiple stock info and keys from quote_data (prompt if blank)
+        """
+            Returns multiple stock info and keys from quote_data (prompt if blank)
 
-        Args:
-            stock (str): stock ticker (or tickers separated by a comma)
-            , prompt if blank
-            key (str): key attributes that the function should return
+            Args:
+                stock (str): stock ticker (or tickers separated by a comma)
+                , prompt if blank
+                key (str): key attributes that the function should return
 
-        Returns:
-            (:obj:`list`): Returns values from each stock or empty list
-                           if none of the stocks were valid
+            Returns:
+                (:obj:`list`): Returns values from each stock or empty list
+                               if none of the stocks were valid
 
         """
+
         #Creates a tuple containing the information we want to retrieve
         def append_stock(stock):
             keys = key.split(',')
             myStr = ''
             for item in keys:
                 myStr += stock[item] + ","
             return (myStr.split(','))
+
         #Prompt for stock if not entered
         if not stock:   #pragma: no cover
             stock = input("Symbol: ")
+
         data = self.quote_data(stock)
         res = []
+
         # Handles the case of multple tickers
         if stock.find(',') != -1:
             for stock in data['results']:
-                if stock == None:
+                if stock is None:
                     continue
                 res.append(append_stock(stock))
+
         else:
             res.append(append_stock(data))
+
         return res
 
+
     def get_quote(self, stock=''):
-        """wrapper for quote_data"""
+        """
+            Wrapper for quote_data
+        """
+
         data = self.quote_data(stock)
         return data["symbol"]
 
-    def get_historical_quotes(
-            self,
-            stock,
-            interval,
-            span,
-            bounds=Bounds.REGULAR
-        ):
-        """fetch historical data for stock
 
-        Note: valid interval/span configs
-            interval = 5minute | 10minute + span = day, week
-            interval = day + span = year
-            interval = week
-            TODO: NEEDS TESTS
+    def get_historical_quotes(self, stock, interval, span, bounds=Bounds.REGULAR):
```

Comment:
>2 args are split up b/c it's easier to add/remove/comment args in debug

Reply:
Debatable because line length is still under 100 characters and it's fairly easy to read/see the parameters and default parameters

Reply:
consistency is king. It's an easy rule to follow, and nearly every other >2 arg function looks one way.